volatility/cli/__init__.py

```diff
@@ -311,8 +314,9 @@ def run(self):
 
         try:
             # Construct and run the plugin
+            options = []
             if constructed:
-                renderers[args.renderer]().render(constructed.run())
+                renderer(options).render(constructed.run())
```

Comment:
I took a look at the implementation of this renderer option and would like to ask although the renderer options has been added globally, the value of the option cannot be passed to the corresponding renderer. Do I need to pass in parameters for the options here?

Reply:
No, you shouldn't have to change the CLI code, you need to define your options within the renderer, as an example you can see:
https://github.com/volatilityfoundation/volatility3/pull/370/files#diff-7360151e7175c6a777de1f856eb59bb2f4937d47cbd6c0198f0a675457165543R150

That should then show up as an `--<renderer_name>-<option_name>`

Reply:
Yes, but something is wrong. I used --<renderer_name>-<option_name>, it didn't work (e.x,:--pretty-separator), and I didn't find the option value being passed to the value in the RenderOption class, did I do something wrong?

Reply:
Hmmm, I'll check it tomorrow and make sure it works...  Happy New Year, by the way!  5:)

Reply:
Thanks a lot, The same to you,Happy New Year!

Reply:
You were quite right, I completely neglected to connect the standard options together with their render options.  I've added that in now, so please let me know if that helps and we can mark this comment as resolved.  5;)

Reply:
I think the problem has been solved, thanks again！